### ISSUE TITLE
Controller issues

### ISSUE THREAD

**Issue by rhobiusvoid** [hrydgard/ppsspp#10876]
This is with 1.5.4 and the latest dev build.
On a few certain games, mainly the Monster Hunter games, I have a weird controller issue. I tested to see if I could replicate this issue, and seems moving in circles has a high chance of triggering it. The issue is, there are moments will move in the direction the analog stick registered for several seconds, and PPSSPP will not accept any further input until after 3-4 seconds, it finally allows me to input again. This issue has been extremely frustrating, and I think I can safely blame it on my system.

**Comment by hrydgard** [comment]
Do you see this with other games on your system? Did it not happen in 1.4.2? 

Any details you can share about your system?

**Comment by ReilerTheDragonknight** [comment]
It never happened on 1.4.2. Here's a full list of the games that this happens in, along with varied "ratings"
Soul Calibur Broken Destiny: Not annoying, but definitely an issue when trying to fight..
Kingdom Hearts: Birth By Sleep Final Mix: Happens constantly, and for a few seconds at a time.
MHF1/FU/P3rd: Happens mainly in town, and is pretty annoying when trying to smith weapons.
Armored Core Last Raven: Very rarely does the view reticle lose control.
Details about my system are as follow:
Windows 7 SP1
AMD Athlon II 170u
Nvidia GeForce 6150 nForce 430
2GB RAM.
The only thing I can say reliably about system is that it's pretty old. Very old. However, I lack the money to upgrade, and this issue never struck me at any time.

**Comment by leopard20** [comment]
@reilerthedragonknight
Do you use wireless controllers? I sometimes do get the issue you mentioned on my machine too (Win 8.1, Corei7 CPU). And it happens in all emulators I tried (PCSX2, Dolphin, etc).

**Comment by ReilerTheDragonknight** [comment]
No, I use a PS3 controller using BetterDS3 to convert it to XInput instead of DInput. However, it only specifically happens on PPSSPP, as emulators like PCSX2 work fine.

**Comment by unknownbrackets** [comment]
Hmm, it is a single core CPU.  We do controller processing on a separate thread - maybe it's not getting enough cycles often enough...

-[Unknown]

**Comment by ReilerTheDragonknight** [comment]
I'm not sure how it came to be an issue like this, since this never usually happened. A CPU can't just expire, can it?

It's weird though. It's really only a problem for a few more taxing games like KH:BBS, but for MHP3rd, I simply just went ahead and got the Non-HD version, and I haven't had problems since. So I think PPSSPP is starting to get too strong for my system. Thanks for tending to the issue anyways, hopefully you guys can figure something out.

So, we were able to single out the problem to be a Single-core issue, and the controls not getting enough cycles to be processed. Wonder how that can be solved...
